jenkins/ci.suse.de/ccp.yaml

```diff
@@ -0,0 +1,66 @@
+- job:
+    name: 'ccp-full-periodic'
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: ccp_repo
+          default: https://github.com/SUSE-Cloud/socok8s.git
+          description: The git repository to use
+      - string:
+          name: PREFIX
+          default: foctodoodle
+          description: The PREFIX to use for openstack resources
+      - string:
+          name: OS_CLOUD
+          default: engcloud
+          description: The name of the OpenStack cloud in clouds.yaml
+      - string:
+          name: KEYNAME
+          default: foctodoodle-key
+          description: The ssh-keyname used to configure nodes and connect.
+      - string:
+          name: INTERNAL_SUBNET
+          default: foctodoodle-subnet
+          description: The subnet to put the machines on.
+
+    triggers:
+      - timed: '0 0 * * *'
+
+    wrappers:
+      - workspace-cleanup:
+          dirmatch: true
+
+    logrotate:
+      numToKeep: 300
+      daysToKeep: -1
+
+    builders:
+      - shell: |
+          set -ex
+          export PREFIX=${PREFIX}
+          export OS_CLOUD=${OS_CLOUD}
+          export KEYNAME=${KEYNAME}
+          export INTERNAL_SUBNET=${INTERNAL_SUBNET}
+
+          # Make sure the job has a cloud available, and environemnt
+          # vars are properly defined
+          cat ~/.config/openstack/clouds.yaml
+          env
+
+          # Get started
+          mkdir ~/ccp/
+          pushd ~/ccp
+              # No need for branchname, as the full periodic job always tests
+              # latest master branch
+              git clone --recursive ${ccp_repo} socok8s
+              pushd socok8s
+                  ./run.sh
+              popd
+          rm -rf socok8s
+          popd
+      - shell: |
+          # Destroy me the CCP env
+          pushd ~/ccp/socok8s
```

Comment:
Not sure if the exports from the previous ```shell``` builder is still there. Maybe you need to export ```PREFIX``` again here?